### ISSUE TITLE
I can not see the options in firefox

### ISSUE THREAD

**Issue by Guillote77** [RandomEngy/RehostImage#4]
this is the error

Error de lectura XML: entidad no definida
Ubicación: chrome://rehostimage/content/options.xul
Número de línea 90, columna 47:		  <description id="dropfileAnonDescription">&dropfile.temphosting.message;</description>
------------------------------------------------------------^

Some way to solve it, thanks

**Comment by danmun** [comment]
Maybe it's because you are using Spanish locale and that message is not in the spanish localization file.
Otherwise I don't know what would cause that error.

**Comment by RandomEngy** [comment]
Yeah someone contributed a new feature and it looks like the Spanish localization is missing some values and the system doesn't handle fallbacks gracefully. So if you supply the translations I can fix this. You can find the files to translate in the source.

**Comment by danmun** [comment]
@Guillote77 find the files here https://github.com/RandomEngy/RehostImage/tree/master/rehostimage/chrome/locale/en-US

**Comment by Guillote77** [comment]
I reinstall firefox in English 64 bits. I did not understand what I had to translate, apologies

**Comment by RandomEngy** [comment]
Translate the files in here:

https://github.com/RandomEngy/RehostImage/tree/master/rehostimage/chrome/locale/en-US

The translations need to be added to these files:

https://github.com/RandomEngy/RehostImage/tree/master/rehostimage/chrome/locale/es-ES
